### PR TITLE
*: Add a empty table for triggers

### PR DESCRIPTION
**Pull request by shenli**
Some admin tool will complain about this.

### CODE REVIEW

File: executor/aggregate_test.go [hunk 1]
```diff
@@ -269,7 +269,7 @@ func (s *testSuite) TestAggregation(c *C) {
 	result.Check(testkit.Rows("<nil>", "<nil>"))
 
 	result = tk.MustQuery("select count(*) from information_schema.columns")
-	result.Check(testkit.Rows("535"))
+	result.Check(testkit.Rows("557"))
```
Comment (ngaut) [inline executor/aggregate_test.go]: Can we avoid hard code number ?

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
@ngaut PTAL

File: executor/aggregate_test.go [hunk 2]
```diff
@@ -269,7 +269,9 @@ func (s *testSuite) TestAggregation(c *C) {
 	result.Check(testkit.Rows("<nil>", "<nil>"))
 
 	result = tk.MustQuery("select count(*) from information_schema.columns")
-	result.Check(testkit.Rows("535"))
+	// When adding new memory table in information_schema, please update this variable.
```
Comment (ngaut) [inline executor/aggregate_test.go]: I mean can we count it dynamically, how about sum something like len(tableTriggersCols) ?

Comment (shenli) [inline executor/aggregate_test.go]: No, we need to get all the tables from infoschema and count their columns.  https://github.com/pingcap/tidb/blob/master/infoschema/tables.go#L421

**Comment by ngaut** [comment]
LGTM

**Pull request merged**